modules/ROOT/pages/design-publish.adoc

```diff
@@ -1,59 +1,64 @@
+When you want to share your API specification with other developers, you can publish it to Anypoint Exchange. 
 
 == Before You Begin
 
-It is a good practice to remove unreferenced files before publishing a project to Anypoint Exchange. An unreferenced file is a file that is not referenced either directly or indirectly from the project's root file.
+It is a good practice to remove unreferenced files before publishing a project to Anypoint Exchange. An _unreferenced_ file is not referenced from the project's root file, either directly or indirectly.
 
-For example, suppose that the file example.raml is among the files in a project. If the root file of the project does not reference the file with any `!include` tag, then the root file doesn't reference the file directly. Also, if no file, in a chain of references from the root file, itself references example.raml, then example.raml is not referenced indirectly from the root file. Therefore, example.raml is unreferenced and should probably not be among the files in your project.
+For example, if the file `example.raml` is among the files in a project but the root file of the project does not reference `example.raml` with an `!include` tag, the root file doesn't reference `example.raml` directly. If none of the files in the chain of references from the root file directly references `example.raml`, it is not referenced indirectly from the root file. Therefore, `example.raml` is unreferenced and should probably not be among the files in the project.
 
+If you do not remove them before you begin, unreferenced files are detected when you select the option to publish to Exchange, and you are given options on whether or not to publish them. 
 
-== Procedure
+[publish-api-specs]
+== Publish an API Specification
 
-. Open the API-specification project that you want to publish, if it is not already open.
-. In the top-right corner of API Designer, click *Publish* and then click *Publish to Exchange*.
+. In API Designer, open the API-specification project that you want to publish.
+. If the specification has dependencies, click the refresh button beside `Exchange Dependencies` to ensure that Design Center has the latest status and content from all dependencies. 
+. Click *Publish* and then click *Publish to Exchange*.
 +
-*Result*: This dialog is opened.
+image:apid-publish-dialog-api-specs.png[The publish dialog shown when you publish from API Designer to Exchange]
 +
-image:apid-publish-dialog-api-specs.png[]
+<1> *Asset version*: The version number to be displayed in the *Version* section of the page that Anypoint Exchange creates for the project. For a description of this field, see <<asset-version>>.
+<2> *API version*: The version number of the API specification. For a description of this field, see <<api-version>>.
+<3> *LifeCycle State*: The lifecycle state of the asset. Assets can be published from API Designer as *In Development* or *Stable*. Assets cannot be deprecated from API Designer. 
```

Comment:
@bjoson-mulesoft one thing that was changed by the UX team together with Rithisha (PM for this initiative) is that `In development` should be replaced by `Development`, since `Development` is used across Exchange. I've fixed this some days ago, so you will maybe need to update the publish modal image and also the wording in the docs. Sorry for the short notice but this was a last minute change as a result of the UX Heuristic reviews. Let me know if I can help you in any way